lib/commands/kendoui-notes.ts

```diff
@@ -0,0 +1,68 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import {KendoUIBaseCommand} from "./kendoui-base";
+
+class KendoUINotesCommand extends KendoUIBaseCommand implements ICommand {
+
+	constructor(private $logger: ILogger,
+		private $opener: IOpener,
+		private $prompter: IPrompter,
+		$errors: IErrors,
+		$kendoUIService: IKendoUIService,
+		$loginManager: ILoginManager,
+		$options: IOptions,
+		$project: Project.IProject) {
+
+		super($errors, $project, $kendoUIService, $loginManager, $options);
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+
+	canExecute(args: string[]): IFuture<boolean> {
+		return ((): boolean => {
+			let number = args ? parseInt(args[0]) : true;
```

Comment:
`let number = args && args[0] ? parseInt(args[0]) : true;`
args is empty array which leads to parseInt(undefined), which leads to: 
Error: The argument undefined is not applicable to this command